test/gui/tst_syncing/test.feature

```diff
@@ -64,26 +26,7 @@ Feature: Syncing files
 
     Scenario: Syncing a file from the server and creating a conflict
         Given user "Alice" has been created on the server with default attributes
-        And user "Alice" has set up a client with these settings and password "1234":
-            """
-            [Accounts]
-            0\Folders\1\ignoreHiddenFiles=true
-            0\Folders\1\localPath=%client_sync_path%
-            0\Folders\1\paused=false
-            0\Folders\1\targetPath=/
-            0\Folders\1\version=2
-            0\Folders\1\virtualFilesMode=off
-            0\dav_user=alice
-            0\display-name=Alice
-            0\http_oauth=false
-            0\http_user=alice
-            0\url=%local_server%
-            0\user=Alice
-            0\version=1
-            version=2
-            [ownCloud]
-            remotePollInterval=5000
-            """
+		And user "Alice" has set up a client with poll interval settings and password "1234"
```

Comment:
```suggestion
		And user "Alice" has set up a client with default settings and poll interval of 5000
```
As for password we can have a global default user values and use one from there.

Reply:
Done